label_studio_sdk/project.py

```diff
@@ -618,8 +629,10 @@ def get_paginated_tasks(
         if only_ids:
             params['include'] = 'id'
 
-        response = self.make_request(
-            'GET', '/api/dm/tasks', params)
+        try:
+            response = self.make_request('GET', '/api/tasks', params)
```

Comment:
this should be released only after corresponding LS/LSE release?

Reply:
Yes.